website/public/.htaccess

```diff
@@ -1,8 +1,15 @@
 # Copyright (C) 2020  Matthew "strager" Glazar
 # See end of file for extended copyright information.
 
-# Optimize loads of the main page and press release pages using HTTP/2 push.
 <IfModule http2_module>
+  Header always set Strict-Transport-Security "max-age=63072000; includeSubDomains"
+  Header always set X-Frame-Options "SAMEORIGIN"
```

Comment:
I'm not sure if we want to prevent iframe embedding.

What security problems does this setting address?

Reply:
We need the `X-Frame-Options` to get an A grade in the scan